### ISSUE TITLE
Screen curtain error

### ISSUE THREAD

**Issue by KeithRempel** [mahmoodhozhabri/VisionAssistantPro#248]
Hi,
Currently if this addon is run with screen curtain enabled it sends a black image to the AI. It would be good if NVDA could give an error message in this instance, to prevent wasting of AI tokens.

**Comment by mahmoodhozhabri** [comment]
Hi @KeithRempel,

Thank you for pointing this out and opening this issue!

This is a very valid point. Sending completely black screenshots to the AI when the Screen Curtain is active is indeed a waste of both user API tokens and network bandwidth.

To resolve this, I have added a check to detect if the Screen Curtain is active (regardless of whether it was toggled permanently via settings or temporarily using the shortcut `NVDA+Control+Escape`).

Now, if a user tries to trigger any visual feature while the Screen Curtain is enabled, the addon will immediately abort the operation and display a helpful, localized error message: *"The Screen Curtain is currently enabled. Please disable it (NVDA+Control+Escape) before using visual recognition features."*

This fix has been implemented and will be published in the next release of the addon. I appreciate your feedback in making this addon more robust!
